<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logowanie_okno</class>
 <widget class="QWidget" name="logowanie_okno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>378</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Loginpage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>SignIn</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_username"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Hasło</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_password">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_login">
              <property name="text">
               <string>Zaloguj się</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_register">
              <property name="text">
               <string>Zarejestruj się</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Registerpage">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1" rowspan="2" colspan="4">
        <widget class="QLineEdit" name="lineEdit_3_login"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Imię</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Nazwisko</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEdit_4_haslo"/>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEdit_1_imie"/>
       </item>
       <item row="7" column="1" colspan="4">
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item row="6" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEdit_5_powtorz_haslo"/>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QLineEdit" name="lineEdit_2_nazwisko"/>
       </item>
       <item row="3" column="0" rowspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Hasło</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" rowspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Powtórz hasło</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Wybierz dział firmy</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2" colspan="3">
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="text">
          <string>Anuluj</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_register_2">
         <property name="text">
          <string>Zarejestruj się</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
